Documentation/observability/hubble-exporter.rst

```diff
@@ -235,8 +235,8 @@ You can change flow log settings without a need for pod to be restarted
       --set hubble.export.dynamic.enabled=true \\
       --set hubble.export.dynamic.config.content[0].name=system \\
       --set hubble.export.dynamic.config.content[0].filePath=/var/run/cilium/hubble/events-system.log \\
-      --set hubble.export.dynamic.config.content[0].IncludeFilters[0].source_pod[0]='kube_system/*' \\
-      --set hubble.export.dynamic.config.content[0].IncludeFilters[1].destination_pod[0]='kube_system/*'
+      --set hubble.export.dynamic.config.content[0].includeFilters[0].source_pod[0]='kube_system/*' \\
+      --set hubble.export.dynamic.config.content[0].includeFilters[1].destination_pod[0]='kube_system/*'
```

Comment:
We can use this opportunity to also fix asterisk (`*`). Either remove it (as this filter prefix matches ns+pod_name) or escape it (`\*`) so the lower line won't be in italics.

Reply:
done